src/gauravk/Assignment_4/AsciiOfCharacter.java

```diff
@@ -0,0 +1,29 @@
+package gauravk.Assignment_4;
+/*#4
+
+Program 1:  Write a method which will return ascii value of given character.
+Hint : Method signature should be int getAsciiOfChar(char ch)
+*/
+
+import java.util.Scanner;
+
+public class AsciiOfCharacter {
+	int asci;
```

Comment:
Ideally, we should create variable as instance when it will be useful at multiple places else we should make it local variable.  this variable **asci** is used in **getAsciiOfChar** this method. 

this does not have impact on logic but just trying to build concepts more clear.